diff -ru a/src/server/animationcurve.h b/src/server/animationcurve.h
--- a/src/server/animationcurve.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/animationcurve.h	2014-02-09 07:38:57.438348783 +0200
@@ -68,14 +68,14 @@
 
 class InQuadCurve : public AnimationCurve {
 public:
-    virtual float value(float f) override {
+    virtual float value(float f) {
         return f * f;
     }
 };
 
 class InOutQuadCurve : public AnimationCurve {
 public:
-    virtual float value(float f) override {
+    virtual float value(float f) {
         f *= 2.f;
         if (f < 1.f) {
             return f * f/ 2.f;
@@ -98,7 +98,7 @@
 
 class OutBackCurve : public BackCurve {
 public:
-    virtual float value(float t) override {
+    virtual float value(float t) {
         float s = m_overshoot;
         t -= 1.f;
         return t*t*((s+1)*t+ s) + 1;
@@ -108,7 +108,7 @@
 
 class InOutBackCurve : public BackCurve {
 public:
-    virtual float value(float t) override {
+    virtual float value(float t) {
         float s = m_overshoot;
         t *= 2.0;
         if (t < 1) {
@@ -124,7 +124,7 @@
 
 class OutBounceCurve : public AnimationCurve {
 public:
-    virtual float value(float t) override {
+    virtual float value(float t) {
         float c = 1.f;
         float a = 0.5f;
         if (t < (4/11.0)) {
@@ -155,7 +155,7 @@
 
 class OutElasticCurve : public ElasticCurve {
 public:
-    virtual float value(float t) override {
+    virtual float value(float t) {
         if (t == 0.f) return 0.f;
         if (t == 1.f) return 1.f;
 
@@ -179,7 +179,7 @@
     PulseCurve() { m_pulseNormalize = 1.f; m_pulseNormalize = 1.f / pulse(1); }
 
     // viscous fluid with a pulse for part and decay for the rest
-    virtual float value(float x) override {
+    virtual float value(float x) {
         if (x >= 1) return 1;
         if (x <= 0) return 0;
 
diff -ru a/src/server/effect.h b/src/server/effect.h
--- a/src/server/effect.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/effect.h	2014-02-09 07:39:09.806410113 +0200
@@ -51,7 +51,7 @@
     {
     protected:
         Settings();
-        virtual std::list<Option> options() const override;
+        virtual std::list<Option> options() const;
     };
 
 private:
diff -ru a/src/server/effects/fademovingeffect.h b/src/server/effects/fademovingeffect.h
--- a/src/server/effects/fademovingeffect.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/effects/fademovingeffect.h	2014-02-09 07:40:32.258818972 +0200
@@ -33,8 +33,8 @@
         Settings();
         ~Settings();
 
-        virtual void unSet(const std::string &name) override;
-        virtual void set(const std::string &name, int v) override;
+        virtual void unSet(const std::string &name);
+        virtual void set(const std::string &name, int v);
 
     private:
         FadeMovingEffect *m_effect;
diff -ru a/src/server/effects/griddesktops.cpp b/src/server/effects/griddesktops.cpp
--- a/src/server/effects/griddesktops.cpp	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/effects/griddesktops.cpp	2014-02-09 07:40:06.666692067 +0200
@@ -26,7 +26,7 @@
 #include "binding.h"
 
 struct DGrab : public ShellGrab {
-    void focus() override
+    void focus()
     {
         if (moving) {
             return;
@@ -39,7 +39,7 @@
             weston_pointer_set_focus(pointer(), view, sx, sy);
         }
     }
-    void motion(uint32_t time, wl_fixed_t x, wl_fixed_t y) override
+    void motion(uint32_t time, wl_fixed_t x, wl_fixed_t y)
     {
         weston_pointer_move(pointer(), x, y);
 
@@ -79,7 +79,7 @@
             }
         }
     }
-    void button(uint32_t time, uint32_t button, uint32_t state) override
+    void button(uint32_t time, uint32_t button, uint32_t state)
     {
         if (state == WL_POINTER_BUTTON_STATE_PRESSED) {
             ShellSurface *shsurf = pointer()->focus ? Shell::getShellSurface(pointer()->focus->surface) : nullptr;
diff -ru a/src/server/effects/griddesktops.h b/src/server/effects/griddesktops.h
--- a/src/server/effects/griddesktops.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/effects/griddesktops.h	2014-02-09 07:40:20.978763037 +0200
@@ -36,10 +36,10 @@
         Settings();
         ~Settings();
 
-        virtual std::list<Option> options() const override;
-        virtual void unSet(const std::string &name) override;
-        virtual void set(const std::string &name, int v) override;
-        virtual void set(const std::string &name, const Option::BindingValue &v) override;
+        virtual std::list<Option> options() const;
+        virtual void unSet(const std::string &name);
+        virtual void set(const std::string &name, int v);
+        virtual void set(const std::string &name, const Option::BindingValue &v);
 
     private:
         GridDesktops *m_effect;
diff -ru a/src/server/effects/inoutsurfaceeffect.h b/src/server/effects/inoutsurfaceeffect.h
--- a/src/server/effects/inoutsurfaceeffect.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/effects/inoutsurfaceeffect.h	2014-02-09 07:39:55.574637065 +0200
@@ -31,8 +31,8 @@
         Settings();
         ~Settings();
 
-        virtual void unSet(const std::string &name) override;
-        virtual void set(const std::string &name, int v) override;
+        virtual void unSet(const std::string &name);
+        virtual void set(const std::string &name, int v);
 
     private:
         InOutSurfaceEffect *m_effect;
diff -ru a/src/server/effects/minimizeeffect.h b/src/server/effects/minimizeeffect.h
--- a/src/server/effects/minimizeeffect.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/effects/minimizeeffect.h	2014-02-09 07:40:13.994728405 +0200
@@ -31,8 +31,8 @@
         Settings();
         ~Settings();
 
-        virtual void unSet(const std::string &name) override;
-        virtual void set(const std::string &name, int v) override;
+        virtual void unSet(const std::string &name);
+        virtual void set(const std::string &name, int v);
 
     private:
         MinimizeEffect *m_effect;
@@ -42,8 +42,8 @@
     ~MinimizeEffect();
 
 protected:
-    virtual void addedSurface(ShellSurface *surf) override;
-    virtual void removedSurface(ShellSurface *surf) override;
+    virtual void addedSurface(ShellSurface *surf);
+    virtual void removedSurface(ShellSurface *surf);
 
 private:
     struct Surface;
diff -ru a/src/server/effects/scaleeffect.cpp b/src/server/effects/scaleeffect.cpp
--- a/src/server/effects/scaleeffect.cpp	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/effects/scaleeffect.cpp	2014-02-09 07:40:00.538661680 +0200
@@ -45,7 +45,7 @@
 };
 
 struct Grab : public ShellGrab {
-    void focus() override
+    void focus()
     {
         Workspace *currWs = Shell::instance()->currentWorkspace();
 
@@ -74,7 +74,7 @@
             tr->alphaAnim.run(tr->surface->output(), ALPHA_ANIM_DURATION);
         }
     }
-    void button(uint32_t time, uint32_t button, uint32_t state) override
+    void button(uint32_t time, uint32_t button, uint32_t state)
     {
         if (state == WL_POINTER_BUTTON_STATE_PRESSED) {
             ShellSurface *shsurf = Shell::getShellSurface(surface->surface);
diff -ru a/src/server/effects/scaleeffect.h b/src/server/effects/scaleeffect.h
--- a/src/server/effects/scaleeffect.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/effects/scaleeffect.h	2014-02-09 07:39:50.238610605 +0200
@@ -36,10 +36,10 @@
         Settings();
         ~Settings();
 
-        virtual std::list<Option> options() const override;
-        virtual void unSet(const std::string &name) override;
-        virtual void set(const std::string &name, int v) override;
-        virtual void set(const std::string &name, const Option::BindingValue &v) override;
+        virtual std::list<Option> options() const;
+        virtual void unSet(const std::string &name);
+        virtual void set(const std::string &name, int v);
+        virtual void set(const std::string &name, const Option::BindingValue &v);
 
     private:
         ScaleEffect *m_effect;
diff -ru a/src/server/effects/zoomeffect.h b/src/server/effects/zoomeffect.h
--- a/src/server/effects/zoomeffect.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/effects/zoomeffect.h	2014-02-09 07:40:26.342789636 +0200
@@ -28,10 +28,10 @@
         Settings();
         ~Settings();
 
-        virtual std::list<Option> options() const override;
-        virtual void unSet(const std::string &name) override;
-        virtual void set(const std::string &name, int v) override;
-        virtual void set(const std::string &name, const Option::BindingValue &v) override;
+        virtual std::list<Option> options() const;
+        virtual void unSet(const std::string &name);
+        virtual void set(const std::string &name, int v);
+        virtual void set(const std::string &name, const Option::BindingValue &v);
 
     private:
         ZoomEffect *m_effect;
diff -ru a/src/server/shellsignal.h b/src/server/shellsignal.h
--- a/src/server/shellsignal.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/shellsignal.h	2014-02-09 07:39:15.434438020 +0200
@@ -54,7 +54,7 @@
     class FunctionFunctor : public Functor {
     public:
         FunctionFunctor(const std::function<void (Args...)> &func) : m_func(func) {}
-        virtual void call(Args... args) override { m_func(args...); }
+        virtual void call(Args... args) { m_func(args...); }
 
         std::function<void (Args...)> m_func;
     };
diff -ru a/src/server/shellsurface.cpp b/src/server/shellsurface.cpp
--- a/src/server/shellsurface.cpp	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/shellsurface.cpp	2014-02-09 07:38:40.846266507 +0200
@@ -482,7 +482,7 @@
 class MoveGrab : public ShellGrab
 {
 public:
-    void motion(uint32_t time, wl_fixed_t x, wl_fixed_t y) override
+    void motion(uint32_t time, wl_fixed_t x, wl_fixed_t y)
     {
         weston_pointer_move(pointer(), x, y);
 
@@ -496,7 +496,7 @@
         weston_view_set_position(view, dx, dy);
         weston_compositor_schedule_repaint(shsurf->m_surface->compositor);
     }
-    void button(uint32_t time, uint32_t button, uint32_t state_w) override
+    void button(uint32_t time, uint32_t button, uint32_t state_w)
     {
         enum wl_pointer_button_state state = (wl_pointer_button_state)state_w;
 
@@ -540,7 +540,7 @@
 class ResizeGrab : public ShellGrab
 {
 public:
-    void motion(uint32_t time, wl_fixed_t x, wl_fixed_t y) override
+    void motion(uint32_t time, wl_fixed_t x, wl_fixed_t y)
     {
         weston_pointer_move(pointer(), x, y);
 
@@ -570,7 +570,7 @@
 
         shsurf->m_client->send_configure(shsurf->m_surface, (int)edges, w, h);
     }
-    void button(uint32_t time, uint32_t button, uint32_t state) override
+    void button(uint32_t time, uint32_t button, uint32_t state)
     {
         if (pointer()->button_count == 0 && state == WL_POINTER_BUTTON_STATE_RELEASED) {
             shsurf->m_runningGrab = nullptr;
diff -ru a/src/server/wl_hawaii/desktop-shell.cpp b/src/server/wl_hawaii/desktop-shell.cpp
--- a/src/server/wl_hawaii/desktop-shell.cpp	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/wl_hawaii/desktop-shell.cpp	2014-02-09 07:39:31.814519245 +0200
@@ -86,7 +86,7 @@
         dismissed = true;
     }
 
-    void focus() override
+    void focus()
     {
         wl_fixed_t sx, sy;
         weston_view *pickedView = weston_compositor_pick_view(pointer()->seat->compositor,
@@ -98,7 +98,7 @@
             weston_pointer_set_focus(pointer(), view, sx, sy);
     }
 
-    void motion(uint32_t time, wl_fixed_t x, wl_fixed_t y) override
+    void motion(uint32_t time, wl_fixed_t x, wl_fixed_t y)
     {
         ShellGrab::motion(time, x, y);
 
@@ -110,7 +110,7 @@
         }
     }
 
-    void button(uint32_t time, uint32_t button, uint32_t state) override
+    void button(uint32_t time, uint32_t button, uint32_t state)
     {
         wl_resource *resource;
         wl_resource_for_each(resource, &pointer()->focus_resource_list) {
@@ -504,7 +504,7 @@
 }
 
 struct BusyGrab : public ShellGrab {
-    void focus() override
+    void focus()
     {
         wl_fixed_t sx, sy;
         weston_view *view = weston_compositor_pick_view(pointer()->seat->compositor, pointer()->x, pointer()->y, &sx, &sy);
@@ -513,7 +513,7 @@
             delete this;
         }
     }
-    void button(uint32_t time, uint32_t button, uint32_t state) override
+    void button(uint32_t time, uint32_t button, uint32_t state)
     {
         weston_seat *seat = pointer()->seat;
 
@@ -1567,7 +1567,7 @@
         return static_cast<DesktopShell *>(Shell::instance());
     }
 
-    virtual std::list<Option> options() const override
+    virtual std::list<Option> options() const
     {
         std::list<Option> list;
         list.push_back(Option::binding("move_window", Binding::Type::Button));
@@ -1579,7 +1579,7 @@
         return list;
     }
 
-    virtual void unSet(const std::string &name) override
+    virtual void unSet(const std::string &name)
     {
         if (name == "move_window") {
             shell()->m_moveBinding->reset();
@@ -1596,7 +1596,7 @@
         }
     }
 
-    virtual void set(const std::string &name, const Option::BindingValue &v) override
+    virtual void set(const std::string &name, const Option::BindingValue &v)
     {
         if (name == "move_window") {
             v.bind(shell()->m_moveBinding);
diff -ru a/src/server/wl_hawaii/desktop-shell.h b/src/server/wl_hawaii/desktop-shell.h
--- a/src/server/wl_hawaii/desktop-shell.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/wl_hawaii/desktop-shell.h	2014-02-09 07:39:37.366546776 +0200
@@ -77,12 +77,12 @@
     void unlockSession();
     void resumeDesktop();
 
-    bool isTrusted(wl_client *client, const char *interface) const override;
+    bool isTrusted(wl_client *client, const char *interface) const;
 
 protected:
     virtual void init();
     virtual void setGrabCursor(Cursor cursor);
-    virtual ShellSurface *createShellSurface(weston_surface *surface, const weston_shell_client *client) override;
+    virtual ShellSurface *createShellSurface(weston_surface *surface, const weston_shell_client *client);
 
 private:
     void idle(void *);
diff -ru a/src/server/wl_hawaii/hawaiiclientwindow.h b/src/server/wl_hawaii/hawaiiclientwindow.h
--- a/src/server/wl_hawaii/hawaiiclientwindow.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/wl_hawaii/hawaiiclientwindow.h	2014-02-09 07:39:43.094575180 +0200
@@ -44,7 +44,7 @@
     void create();
 
 protected:
-    virtual void added() override;
+    virtual void added();
 
 private:
     ShellSurface *shsurf();
diff -ru a/src/server/wl_hawaii/hawaiiworkspace.h b/src/server/wl_hawaii/hawaiiworkspace.h
--- a/src/server/wl_hawaii/hawaiiworkspace.h	2014-02-08 22:19:29.000000000 +0200
+++ b/src/server/wl_hawaii/hawaiiworkspace.h	2014-02-09 07:39:20.926465254 +0200
@@ -48,7 +48,7 @@
     static HawaiiWorkspace *fromResource(wl_resource *resource);
 
 protected:
-    virtual void added() override;
+    virtual void added();
 
 private:
     void activeChanged();
